_headers

```diff
@@ -41,6 +41,7 @@ See Netlify's syntax: https://docs.netlify.com/routing/headers/
   {% assign imgsrc="'self' https://img.shields.io https://www.netlify.com https://simpleicons.org/ https://github.githubassets.com/ " %}
   {% if page.layout == 'product' %}
     Link: /api{{page.permalink}}.json; rel=alternate;type=application/json
+    Link: /calendar{{page.permalink}}.ics; rel=alternate;type=text/calendar
```

Comment:
We can add a CORS header as well, to make it easier to embed in other websites.

Reply:
Note sure it is useful but it does not hurt. Done.